query_optimizer/strategy/tests/Aggregate_unittest.cpp

```diff
@@ -78,54 +82,59 @@ class AggregateTest : public StrategyTest {
 };
 
 TEST_F(AggregateTest, BasicTest) {
-  const E::AggregateFunctionPtr sum = createSum(add_literal_0_, true);
-  const E::AliasPtr sum_alias = createAlias(sum,
-                                            "sum" /* attribute_name */,
-                                            "test_relation" /* relation_name */);
-  input_logical_ = L::Aggregate::Create(
-      logical_table_reference_0_,
-      {alias_0_} /* grouping_expressions */,
-      {sum_alias} /* aggregate_expressions */);
-  expected_physical_ = P::Aggregate::Create(
-      physical_table_reference_0_,
-      {alias_0_} /* grouping_expressions */,
-      {sum_alias} /* aggregate_expressions */,
-      nullptr /* filter_predicate */);
-  EXPECT_CORRECT_PHYSICAL();
+  for (bool is_distinct : {true, false}) {
+    const E::AggregateFunctionPtr sum = createSum(add_literal_0_, true, is_distinct);
+    const E::AliasPtr sum_alias = createAlias(sum,
+                                              "sum" /* attribute_name */,
+                                              "test_relation" /* relation_name */);
```

Comment:
I know this was from the existing code, but why the relation name is a fixed string, instead of a function call to get the name?